docs/source/upcoming_release_notes/1248-DOC:ST1K4_decoupling_ST3K4_when_ST3K4_OUT.rst

```diff
@@ -0,0 +1,30 @@
+1248 DOC:ST1K4 decoupling ST3K4 when ST3K4 OUT
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+- Decoupling ST1K4 with ST3K4 when ST3K4 OUT
```

Comment:
This isn't what the code change does here. The decoupling is implemented in the PLC code.
This also should be under `Device Updates`, not under `Features`.
Maybe something more like this?
```
- Allow ST1K4 to move regardless of auto mode.
  See https://github.com/pcdshub/lcls-plc-kfe-motion/pull/122,
  where we decouple ST1K4 from ST3K4 when ST3K4 is OUT.
```